lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2443,9 +2443,9 @@ def bar3d(self, x, y, z, dx, dy, dz, color=None,
 
         facecolors = []
         if color is None:
-            # no color specified
-            facecolors = [None] * len(x)
-        elif len(color) == len(x):
+            color = self._get_lines.get_next_color()
```

Comment:
also, I don't think it would be `self._get_lines`. That would be for line plots.